bucket/melonds.json

```diff
@@ -10,8 +10,8 @@
         "ATTENTION: MelonDS requires BIOS and firmware files dumped from a Nintendo DS to function.",
         "Visit http://melonds.kuribo64.net/faq.php for more information."
```

Comment:
According to the release notes, a BIOS file is no longer an absolute requirement for DS mode operation (although I assume it works better with a BIOS). It's only required for DSi mode operation.

Reply:
Neat, didn't know that changed